src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -1,16 +1,879 @@
+        );
+        writer.write_f64(
+            "HYD_BLUE_RESERVOIR",
+            self.blue_loop.get_reservoir_volume().get::<gallon>(),
+        );
+        writer.write_bool("HYD_BLUE_EPUMP_ACTIVE", self.blue_electric_pump.is_active());
+
+        writer.write_f64(
+            "HYD_YELLOW_PRESSURE",
+            self.yellow_loop.get_pressure().get::<psi>(),
+        );
+        writer.write_f64(
+            "HYD_YELLOW_RESERVOIR",
+            self.yellow_loop.get_reservoir_volume().get::<gallon>(),
+        );
+        writer.write_bool(
+            "HYD_YELLOW_EDPUMP_ACTIVE",
+            self.engine_driven_pump_2.is_active(),
+        );
+        writer.write_bool(
+            "HYD_YELLOW_FIRE_VALVE_OPENED",
+            self.yellow_loop.get_fire_shutoff_valve_state(),
+        );
+        writer.write_bool(
+            "HYD_YELLOW_EPUMP_ACTIVE",
+            self.yellow_electric_pump.is_active(),
+        );
+
+        writer.write_bool("HYD_PTU_VALVE_OPENED", self.ptu.is_enabled());
+        writer.write_bool("HYD_PTU_ACTIVE_Y2G", self.ptu.get_is_active_right_to_left());
+        writer.write_bool("HYD_PTU_ACTIVE_G2Y", self.ptu.get_is_active_left_to_right());
+        writer.write_f64(
+            "HYD_PTU_MOTOR_FLOW",
+            self.ptu.get_flow().get::<gallon_per_second>(),
+        );
+
+        writer.write_f64("HYD_RAT_STOW_POSITION", self.rat.get_stow_position());
+
+        writer.write_f64("HYD_RAT_RPM", self.rat.prop.get_rpm());
+
+        //BRAKES
+        writer.write_f64(
+            "HYD_BRAKE_NORM_LEFT_PRESS",
+            self.braking_circuit_norm
+                .get_brake_pressure_left()
+                .get::<psi>(),
+        );
+        writer.write_f64(
+            "HYD_BRAKE_NORM_RIGHT_PRESS",
+            self.braking_circuit_norm
+                .get_brake_pressure_right()
+                .get::<psi>(),
+        );
+        writer.write_f64(
+            "HYD_BRAKE_ALTN_LEFT_PRESS",
+            self.braking_circuit_altn
+                .get_brake_pressure_left()
+                .get::<psi>(),
+        );
+        writer.write_f64(
+            "HYD_BRAKE_ALTN_RIGHT_PRESS",
+            self.braking_circuit_altn
+                .get_brake_pressure_right()
+                .get::<psi>(),
+        );
+        writer.write_f64(
+            "HYD_BRAKE_ALTN_ACC_PRESS",
+            self.braking_circuit_altn.get_acc_pressure().get::<psi>(),
+        );
+    }
+}
+
+pub struct A320HydraulicBrakingLogic {
+    parking_brake_demand: bool,
+    weight_on_wheels: bool,
+    left_brake_command: f64,
+    right_brake_command: f64,
+    left_brake_green_command: f64, //Actual command sent to left green circuit
+    left_brake_yellow_command: f64, //Actual command sent to left yellow circuit
+    right_brake_green_command: f64, //Actual command sent to right green circuit
+    right_brake_yellow_command: f64, //Actual command sent to right yellow circuit
```

Comment:
**issue**: Comments next to a variable indicate the variable might not be properly named, or that a newtype might be useful, e.g.:

```rust
struct CommandedBrake???(f64);
```

(I added ??? because based on all the code below I can't understand what this actual command is commanding).